saleor/checkout/views/__init__.py

```diff
@@ -72,7 +72,7 @@ def login(request, checkout):
     Allows user to choose if he wants to login before checkout or continue
     as an anonymous user
     """
-    if request.user.is_authenticated:
+    if request.user.is_authenticated():
```

Comment:
On Django 1.8, where `is_authenticated` is still a method, this will always return `True`, it was incorrectly written at the first time. If we want to keep compatibility, it should stay this way. 
It should be supported till Django 2.0.